common/views/components/Modal/Modal.tsx

```diff
@@ -34,7 +34,7 @@ type Props = {
   openButtonRef?: MutableRefObject<HTMLElement | null>;
   removeCloseButton?: boolean;
   showOverlay?: boolean;
-  modalStyle?: 'filters' | 'calendar' | 'default';
+  modalStyle?: 'filters' | 'calendar' | 'default' | 'filters-new';
```

Comment:
super nitpick; I'd leave default to be the last one.
Although having said that I looked at the switch/case using it 

```
function determineModal(modalStyle: Props['modalStyle']) {
  switch (modalStyle) {
    case 'filters':
      return FiltersModal;
    case 'filters-new':
      return FiltersModalNew;
    case 'calendar':
      return CalendarModal;
    default:
      return BaseModalWindow;
  }
}
```

and it seems like it just ignores the 'default' string (going to default anyway). Is 'default' specified anywhere? Could it just be removed from the allowed types?

Reply:
done